synapse/lib/ast.py

```diff
@@ -2866,6 +2866,9 @@ class FuncCall(Value):
     async def compute(self, runt, path):
 
         func = await self.kids[0].compute(runt, path)
+        if not callable(func):
+            raise s_exc.StormRuntimeError(mesg=f"'{type(func).__name__}' object is not callable.")
```

Comment:
If we have the name of variable here that could be useful to include.

Reply:
We don't always, but after the AST changes are merged we would have the text to include here which may be better